chia/cmds/chia.py

```diff
@@ -39,26 +42,58 @@ def monkey_patch_click() -> None:
     context_settings=CONTEXT_SETTINGS,
 )
 @click.option("--root-path", default=DEFAULT_ROOT_PATH, help="Config file root", type=click.Path(), show_default=True)
+@click.option(
+    "--keys-root-path", default=DEFAULT_KEYS_ROOT_PATH, help="Keyring file root", type=click.Path(), show_default=True
+)
+@click.option("--passphrase-file", type=click.File("r"), help="File or descriptor to read the keyring passphase from")
 @click.pass_context
-def cli(ctx: click.Context, root_path: str) -> None:
+def cli(ctx: click.Context, root_path: str, **kwargs) -> None:
     from pathlib import Path
 
     ctx.ensure_object(dict)
     ctx.obj["root_path"] = Path(root_path)
 
+    keys_root_path = kwargs.get("keys_root_path")
+    if keys_root_path:
+        set_keys_root_path(Path(keys_root_path))
+
+    passphrase_file = kwargs.get("passphrase_file")
+    if passphrase_file:
+        from .passphrase_funcs import cache_passphrase, read_passphrase_from_file
```

Comment:
whats the motivation for doing this here and not with the rest of the imports ?

Reply:
This is done to save time when running other commands. Otherwise the chia command would take a long time to initialize

Reply:
Thanks for the background info on this, Mariano. I thought it might have been to resolve cyclic imports from modules that import chia.py